### DISCUSSION TITLE
How can I configure TinyMCE to remove the class attribute from all elements?

### DISCUSSION THREAD

**Discussion by tanggd** [tinymce/tinymce#9491] [General]
How can I configure TinyMCE to remove the class attribute from all elements?

```js
tinymce.init({
    selector: 'textarea',
    '???': '???',
    // eg:
    'remove_classes_all_elements': true,
});
```

tinymce@6.x

**Comment by lostkeys** [comment]
Hi @tanggd. Have a look at the [valid classes config option](https://www.tiny.cloud/docs/tinymce/latest/content-filtering/#valid_classes).
